#172693257 Fix codeclimate issues

descholar-ceo opened the pull request:
### What does this PR ​ do​ ? 
Fix code climate issues and refactoring services

### Description of Task to be completed?

- [x] Fix test failing
- [x] Refactor .codeclimate.yml file
- [ ] Refactor services
- [ ] Solving conflicts whenever there is a new push to the make-multicity request PR

### How should this be manually tested?
Refer to this PR [#171380754 make multi-city request #29](https://github.com/Stackup-Rwanda/stackup2-barefoot-backend/pull/29)

### Any background context you want to provide?
If you are running [Node 14.x or Node below 12.x](https://nodejs.org/en/), this app might behave strangely, unlikely to work for you

### What are the relevant pivotal tracker stories?
[ #172693257](https://www.pivotaltracker.com/n/projects/2435482/stories/172693257)

### Screenshots 
N/A

### Questions:
N/A

muchristian committed:
feat(User can make multi city request)

create model trip
create trip migration

Muhire-Josue reviewed (commented):
great work, could you squash those commit?

descholar-ceo committed:
bug(requests):Fix codeclimate issues …

@descholar-ceo
descholar-ceo committed 3 hours ago
- Fix test failing
- Refactor services
- Refactor .codeclimate.yml file

[Finishes #172693257]

descholar-ceo commented:
> great work, could you squash those commit?

@Muhire-Josue Which commits?
